# This is the lli input message format for AAUSHIP
uint8 DevID
uint8 MsgID
int16 Data
float64 Time
